### PR TITLE
Document BYOC AWS Managed Resources

### PR DESCRIPTION
**Pull request by jordanstephens**
TODO
- [x] List all resources created by defang in your account for bootstrapping
- [x] List all resources created by defang for each project

### CODE REVIEW

**Comment by jordanstephens** [comment]
@lionello I suspect some of these resources aren't in the right category, so double check my partitioning.

File: docs/providers/aws/aws.md [hunk 1]
```diff
@@ -61,3 +61,85 @@ When using [Managed Postgres](/docs/concepts/managed-storage/managed-postgres.md
...
+*    └─ aws:ec2/subnet:Subnet                                      shared-vpc-public-3
+*       └─ aws:ec2/routeTable:RouteTable                           shared-vpc-public-3
+*          ├─ aws:ec2/routeTableAssociation:RouteTableAssociation  shared-vpc-public-3
+*          └─ aws:ec2/route:Route                                  shared-vpc-public-3
+* aws:iam/role:Role                                                ecs-task-role
+* aws:ec2/networkAcl:NetworkAcl                                    shared-nacl
+* aws:ec2/securityGroup:SecurityGroup                              sg443
+```
+
+Then, defang will manage the following resources for each project you deploy:
```
Comment (lionello) [inline docs/providers/aws/aws.md:111]: Right now, in BYOC we bootstrap each project, so there's no infra shared between projects.

**Review by lionello — CHANGES_REQUESTED** [review]

File: docs/providers/aws/aws.md (hunk 1 above)
Comment (jordanstephens) [reply docs/providers/aws/aws.md:111]: That's what I intended to communicate by saying that the resources were managed "for each project" Is there a clearer way we can say that? Or should we just be extra explicit and add "None of these resources are shared between projects"?
```suggestion
Then, defang will manage the following resources for each project you deploy (None of these resources are shared between projects):
```

File: docs/providers/aws/aws.md [hunk 2]
```diff
@@ -61,3 +61,85 @@ When using [Managed Postgres](/docs/concepts/managed-storage/managed-postgres.md
 ### Managed Redis
 
 When using [Managed Redis](/docs/concepts/managed-storage/managed-redis.md), the Defang CLI provisions an ElastiCache Redis cluster in your account.
+
+### Managed Resources
+
+Defang will create and manage the following resources in your AWS account as part of its bootstrap procedure:
```
Comment (lionello) [inline docs/providers/aws/aws.md:67]: I don’t think this is correct. Here, I expected to see the resources in our CloudFormation stack.

**Review by lionello — CHANGES_REQUESTED** [review]

**Pull request merged**